samples/glean/samples-glean-library/metrics.yaml

```diff
@@ -0,0 +1,19 @@
+# This file defines the metrics that are recorded by glean telemetry. They are
+# automatically converted to Kotlin code at build time using the `glean_parser`
+# PyPI package.
+
+$schema: moz://mozilla.org/schemas/glean/metrics/1-0-0
+
+sample_metrics:
+  test:
+    type: counter
+    description: >
+      A simple counter defined in a third-party library
+    send_in_pings:
+      - baseline
```

Comment:
Since this is an example, let's not put it into the `baseline` ping, to not set a dangerous precedent :) Let's omit this so that it shows up in the `metrics` ping and also call this out in the readme file.

Reply:
Sure.  It's easier to test the baseline ping because it goes out when you press the button -- you don't have to wait 24 hours :)  Maybe we should make the metrics ping go out then, too?

Reply:
Yeah, that's a good idea

Reply:
So, since this is blocked on [this bug](https://bugzilla.mozilla.org/show_bug.cgi?id=1525333), I would still change this to `metrics` and eventually change the sample to trigger the "metrics" ping once that bug settles. Let's not block on this PR for that bug!

Reply:
Ok, we discussed this and Mike suggested a good compromise: let's leave this in the `metrics` ping but add a comment stating that, for testing this manually, it's better for it to be in the `baseline` ping.